frontend/src/pages/create_campaign/Campaign/index.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { FormControlLabel, Switch } from "@mui/material";
+import PropTypes from "prop-types";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import LocalizationProvider from "@mui/lab/LocalizationProvider";
+import DateTimePicker from "@mui/lab/DateTimePicker";
+import Dropzone from "react-dropzone";
+import {
+  CampaignContainer,
+  CampaignDropzone,
+  CampaignRowDiv,
+  SwitchRowDiv,
+  CampaignTextField,
+} from "./campaignTab.styled";
+import { fileToDataUrl } from "../../../utils";
+
+const CampaignTab = ({ isSelected, campaignData }) => {
+  const {
+    campaignName,
+    setCampaignName,
+    startDate,
+    setStartDate,
+    endDate,
+    setEndDate,
+    description,
+    setDescription,
+    interviewStage,
+    setInterviewStage,
+    scoringStage,
+    setScoringStage,
+    cover,
+    setCover,
+  } = campaignData;
+  console.log(campaignData);
+  const onFileUpload = async (acceptedFiles) => {
+    const fileUrl = await fileToDataUrl(acceptedFiles[0]);
+    setCover(fileUrl);
+  };
+
+  return (
+    isSelected && (
+      <CampaignContainer>
+        <Dropzone
+          onDrop={(acceptedFiles) => onFileUpload(acceptedFiles)}
+          accept={["image/jpeg", "image/jpg", "image/png", "image/gif"]}
+          minSize={1024}
+          maxSize={3072000}
+        >
+          {({ getRootProps, getInputProps }) => (
+            <section>
+              <CampaignDropzone {...getRootProps()}>
+                <input {...getInputProps()} />
+                {cover === null && (
+                  <p>
+                    Drag and drop your campaign cover image, or click to select
+                    an image
+                  </p>
+                )}
+                {cover !== null && <img src={cover} alt="campaign cover" />}
```

Comment:
is there styling here for the size of the cover? ideally we want it to remain the same size no matter what pic they upload i think - instead of using a IMG here, i would rather use a div of fixed size and then set `background-image` to the cover https://www.w3schools.com/cssref/pr_background-image.asp

BUt if you cant get it working all g!! as long as the img remains same size - i just know that background img proeprty is easy to use for this purpose